docker-image/README.md

```diff
@@ -11,19 +11,19 @@ We support several base images:
 - [Docker Hub] jiata/thunderbolt:<image-version>-spark2.1.0-python2.7.13
 - [Docker Hub] jiata/thunderbolt:<image-version>-spark1.6.3-python2.7.13
 
-NOTE: Replace **<image-version>** with the version of the thunderbolt image that maps to the version of Azure Thunderbolt that you are using. For example: **jiata/thunderbolt:0.1.0-spark2.2.0-python3.5.4**
+NOTE: Replace **<image-version>** with the version of the image you wish to use. For example: **jiata/thunderbolt:0.1.0-spark2.2.0-python3.5.4**
 
 ### Gallery of 3rd Party Images
-Since Azure Thunderbolt is built with Docker containers, users can bring their own images to use with Azure Thunderbolt:
+Since this toolkit uses with Docker containers to run Spark, users can bring their own images. Here's a list of 3rd party images:
 - *coming soon*
```

Comment:
Did you mean to leave the *coming soon* here?

Reply:
yep. so show that it is coming soon i guess..